cognee/api/v1/cognify/routers/get_cognify_router.py

```diff
@@ -6,22 +6,24 @@
 from cognee.modules.users.methods import get_authenticated_user
 from fastapi import Depends
 
+
 class CognifyPayloadDTO(BaseModel):
     datasets: List[str]
+    graph_model: BaseModel
 
 def get_cognify_router() -> APIRouter:
     router = APIRouter()
 
     @router.post("/", response_model=None)
-    async def cognify(payload: CognifyPayloadDTO, user: User = Depends(get_authenticated_user)):
+    async def cognify(payload: CognifyPayloadDTO, user: User = Depends(get_authenticated_user), graph_model:BaseModel=None):
```

Comment:
If graph_model is part of the CognifyPayloadDTO I don't think we need it as an input parameter in the endpoint.